app-text/mupdf/mupdf-1.21.1.ebuild

```
@@ -154,7 +157,8 @@ src_install() {
 	fi
 
 	# Respect libdir (bug #734898)
-	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die "Failed to sed pkgconfig file to respect libdir in src_install()"
+	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || \
+		die "Failed to sed pkgconfig file to respect libdir in src_install()"
```

nit: the control operator is usually put on the next line
```suggestion
		|| die "Failed to sed pkgconfig file to respect libdir in src_install()"
```

---

Makes sense, fixed.